packages/datagovmy-ui/src/charts/partials/mapplot.tsx

```diff
@@ -76,9 +77,19 @@ const CatalogueMapPlot: FunctionComponent<CatalogueMapPlotProps> = ({
     }),
     [png, ctx]
   );
-  return (
-    <MapPlot _ref={ctx} id={dataset.meta.unique_id} className={className} markers={dataset.chart} />
-  );
+
+  const markers = useMemo<MarkerData>(() => {
+    const _markers: MarkerData = dataset.chart.map((e: any) => {
+      const { position, ...tooltip } = e;
+      return {
+        position: position,
+        tooltip: tooltip,
+      };
+    });
+    return _markers;
+  }, []);
```

Comment:
Pls dont put empty dep array here. You're basically telling it to run once. If the tooltip array changes, it will not re-render. Either drop the useMemo, or put `dataset.chart` as a dependency